yml/Desktop_App/vtso.yml

```diff
@@ -0,0 +1,29 @@
+---
+Name: Microsoft Visual Studio Tools for Office (VSTO)
+Description: VSTO is a functionality available in Visual Studio which allows the user to craft a .NET Office Add-in to be shipped with an office document.
```

Comment:
Modify to:
VSTO is functionality available in Visual Studio, which allows the user to craft a .NET Office Add-in to be shipped with an office document.

Reply:
Done